language-plutus-core/src/Language/PlutusCore/Pretty/Error.hs

```diff
@@ -0,0 +1,69 @@
+-- | Pretty-printing of various types of PLC errors.
+
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
I think pretty-printing is mostly clutter and you want it to be in a separate place. I certainly do not like interleaving important data type definitions and `Pretty*` instances. But if people prefer, I can put them back at the end of the file.

Reply:
I don't hate it, but my expectations in Haskell are that people try and avoid orphans, so if I'm looking for a typeclass implementation I should either look at the class definition site or the type definition site. It's not a huge deal.

Reply:
I'd prefer it. A separate module avoids some mess, but it can lead to big problems if we define overlapping instances - banning orphans makes this impossible.